### ISSUE TITLE
'fvctl dumpConfig' exits with status 0 even if it can't write to the file

### ISSUE THREAD

**Issue by jbsbbn** [opennetworkinglab/flowvisor#157]
Thus:

```
+$ fvctl --passwd-file=/etc/flowvisor.passwd dumpConfig /config.json    
Error: /config.json (Permission denied)

+$ echo $?
0
```

It should exit with a non-zero status, like it does e.g. if you specify an incorrect password file or mistype the password.

**Comment by jbsbbn** [comment]
This affects 0.8.17, but isn't important enough to be worth a new 0.8 release (IMHO); please only fix it in 0.10 and beyond.

**Comment by alshabib** [comment]
XMLRPC interface is now deprecated. So this will not be fixed.
